exes/mqtt-broadcaster/CMakeLists.txt

```diff
@@ -1,16 +1,21 @@
-add_executable(mqtt-broadcaster
+add_executable(mqtt_broadcaster
```

Comment:
Has it not evolved from only being a broadcaster?

Reply:
Yes it has, changing the name to `mqtt_bridge`.